ckanext/cfpb_extrafields/options.py

```diff
@@ -0,0 +1,138 @@
+# resources 
+def format():
+    return ["CSV", "ASCII", "STATA", "SQL", "PDF", "SAS", "XML", "word doc", 
+           "excel", "TXT", "TAB", "JSON"] #+other
+def storage_location():
+    return ["Research Server", "SQLServer", "Postgres", "MS Access", "Z Drive", "SES",
+            "RightNow"] #+other
+def sensitivity_level():
+    a=["Public", "Low", "Medium", "High"]
+    # insert a blank at the beginning
+    a.insert(0,"")
```

Comment:
probably not a huge priority, but it seems like this code seems to repeat a good number of times.  The formatting of a list into the correct form.select options list could be abstracted away pretty easily:
```
def form_select_format(list):
    return [{'value'}: key for key in list.append("")]
```
This would reduce the number of times you have to copy and paste the same comments, and makes it more consistent with the other methods.